pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -51,8 +61,22 @@ type InstallationStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
-	Stages        map[int]string         `json:"stages"`
-	ProductStatus map[ProductName]string `json:"product_status"`
+	Stages map[string]*InstallationStageStatus `json:"stages"`
+}
+
+type InstallationStageStatus struct {
+	Name     string                                     `json:"name"`
+	Phase    StatusPhase                                `json:"phase"`
+	Products map[ProductName]*InstallationProductStatus `json:"products"`
+}
+
+type InstallationProductStatus struct {
+	Name    ProductName    `json:"name"`
+	Version ProductVersion `json:"version"`
+	Host    string         `json:"host"`
+	Type    string         `json:"type,omitempty"`
+	Mobile  bool           `json:"mobile,omitempty"`
```

Comment:
what is this for?

Reply:
no idea, it was just there in the installation repo, so I kept it.